### ISSUE TITLE
Total time for training the DIM dataset

### ISSUE THREAD

**Issue by wuyujack** [Yaoyi-Li/GCA-Matting#3]
Hi Yaoyi,

Thank you for your great work and I hope to discuss it with you in AAAI-20 Technical Program on Feb. if applicable! 

One question is that I am curious about the total time you need for training the whole DIM training set. It seems like you only include the total iterations for the training phase in your paper. 

As I can see that there are so many augmentations based on cv2 without acceleration, the potential clock time issue for training should be in the data loader section. Is that the same observation on your side?

Regards,
Mingfu Liang

**Comment by Yaoyi-Li** [comment]
Thanks for your interest. It takes me more than 2 days to train the model with 4 RTX2080Ti GPUs and around 1s for each batch. The model also converges with less iterations like 100000, but more iterations will slightly improve the performance.

The data augmentations do require much CPU time, but a 512x512 autoencoder is also not very efficient. In our training, the data loader did not introduce a very obvious latency in the GPU training.

**Comment by wrrJasmine** [comment]
hi  Yaoyi,
Thank you for your great work！
I am trying to reproduce your work. But when I train the model, I found that it takes too much time each batch. you mentioned in your experiment , it is about 1s for each batch and 1min each batch for me. I just followed your code. I only use one gpu p100 and batch size is 8. Do you know what's the reason.

best wishes！

**Comment by ucasiggcas** [comment]
hi,dear
how to set the config file?
https://github.com/Yaoyi-Li/GCA-Matting/issues/25

**Comment by 99991** [comment]
To add another point of data: Training took 37 hours with two V100 32GB GPUs. I changed `nproc_per_node=2` in https://github.com/Yaoyi-Li/GCA-Matting/blob/5a999be4b00bb482658fa7013a74a2b8acddc78e/train.sh#L3

**Comment by wrrJasmine** [comment]
你好，邮件已经收到，我会尽快给你回复。This is an automatic reply, confirming that your email was received.I will reply as soon as possible.Thank you
